airgun/entities/subscription.py

```diff
@@ -139,6 +139,19 @@ def read_subscription_manifest_expiration_date_only(self):
             manage_view.close_button.click()
         return manifest_expiration_date
 
+    def is_subscription_manifest_header_message_display(self):
+        """Is header & message present in manage manifest modal box or not"""
```

Comment:
"Checks header and massage present in the manage manifest modal"
Non - blocking